pkg/controller/networkpolicy/validate_test.go

```diff
@@ -1434,18 +1485,53 @@ func TestValidateAntreaPolicy(t *testing.T) {
 					},
 				},
 			},
+			op:             admv1.Create,
 			expectedReason: "layer 7 protocols can only be used when L7NetworkPolicy is enabled",
 		},
+		{
+			name: "update-tier",
+			policy: &crdv1alpha1.ClusterNetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "non-existent-tier",
+				},
+				Spec: crdv1alpha1.ClusterNetworkPolicySpec{
+					AppliedTo: []crdv1alpha1.AppliedTo{
+						{
+							PodSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{"foo": "bar"},
+							},
+						},
+					},
+					Tier: "NetworkOps",
+				},
+			},
+			oldPolicy: &crdv1alpha1.ClusterNetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "non-existent-tier",
+				},
+				Spec: crdv1alpha1.ClusterNetworkPolicySpec{
+					AppliedTo: []crdv1alpha1.AppliedTo{
+						{
+							PodSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{"foo": "bar"},
+							},
+						},
+					},
+					Tier: "non-existent-tier",
+				},
+			},
+			op: admv1.Update,
+		},
```

Comment:
no test cases for delete operation?

Reply:
// Delete of Antrea Policies have no validation. This will be an empty for loop. So no test case for delete